lib/optimization/event_test.go

```diff
@@ -267,6 +267,27 @@ func TestTableData_ShouldFlushRowLength(t *testing.T) {
 	assert.Equal(t, "rows", flushReason)
 }
 
+func TestTableData_ContainsHardDeletes(t *testing.T) {
+	{
+		// Hard delete = true
+		td := NewTableData(nil, nil, kafkalib.TopicConfig{}, "foo")
+		assert.Equal(t, 0, int(td.Rows()))
+
+		td.InsertRow("123", nil, true)
+		assert.Equal(t, 1, int(td.Rows()))
+		assert.True(t, td.containsHardDeletes)
```

Comment:
Call `ContainsHardDeletes()` since we have a method for it?

Reply:
Could also test inserting something that isn't a delete.

Reply:
sg